src/main/java/com/vinylteam/vinyl/exception/entity/PasswordRecoveryError.java

```diff
@@ -15,6 +15,7 @@
 
     TOKEN_NOT_CORRECT_UUID("Your link is incorrect! Please check the link in the your email"),
     TOKEN_NOT_FOUND_IN_DB("Your link is incorrect! Please check the link in the your email or contact support."),
+    DELETE_TOKEN_ERROR("Sorry, problem with processing this link. Please, request new one or contact support."),
```

Comment:
Lets have consistency here, either all constants with _ERROR suffix or none of them